src/linear_algebra/quadratic_form.lean

```diff
@@ -101,7 +101,7 @@ namespace quadratic_form
 variables {Q : quadratic_form R M}
 
 instance : has_coe_to_fun (quadratic_form R M) :=
-⟨_, λ B, B.to_fun⟩
+⟨_, to_fun⟩
```

Comment:
I had a hunch that this was making unification harder for me. At any rate, the new spelling is shorter.